### PR TITLE
updated list of compliance profiles

### PR DESCRIPTION
**Pull request by modrake**
This was requested by a customer; there's probably a better way to format this which I will leave to our docs team, but I wanted to get the data entry part taken care of. Thanks all!

### CODE REVIEW

File: chef_master/source/compliance.rst [hunk 1]
```diff
@@ -59,9 +59,59 @@ Compliance profiles are built and maintained from the Chef Compliance server web
...
+* CIS Apple OSX 10.11 Benchmark Level 2
+* CIS Centos Linux 6 Benchmark Level 1
+* CIS Centos Linux 6 Benchmark Level 2
+* CIS Centos Linux 7 Benchmark Level 1
+* CIS Centos Linux 7 Benchmark Level 2
+* CIS HP-UX 11iv3 Update 2 Benchmark Level 1
+* CIS HP-UX 11iv3 Update 2 Benchmark Level 2
+* CIS Red Hat Enterprise Linux 6 Benchmark Level 1
+* CIS Red Hat Enterprise Linux 6 Benchmark Level 2
+* CIS Red Hat Enterprise Linux 6 Benchmark Level 1
```
Comment (chris-rock) [inline chef_master/source/compliance.rst]: CIS Red Hat Enterprise Linux **7** Benchmark Level 1

File: chef_master/source/compliance.rst [hunk 2]
```diff
@@ -59,9 +59,59 @@ Compliance profiles are built and maintained from the Chef Compliance server web
...
+* CIS Ubuntu 12.04 LTS Server Benchmark Level 2
 * CIS Ubuntu 14.04 LTS Server Benchmark Level 1
 * CIS Ubuntu 14.04 LTS Server Benchmark Level 2
+* CIS Ubuntu 16.04 LTS Server Benchmark Level 1
+* CIS Ubuntu 16.04 LTS Server Benchmark Level 2
+* CIS Microsoft Windows 10 Enterprise (Release 1511) Benchmark Level 1 
+* CIS Microsoft Windows 10 Enterprise (Release 1511) Benchmark Level 1 +BitLocker
+* CIS Microsoft Windows 10 Enterprise (Release 1511) Benchmark Level 2
+* CIS Microsoft Windows 10 Enterprise (Release 1511) Benchmark Level 2 +BitLocker
+* CIS Microsoft Windows Server 2012(non-R2) Benchmark Level 1 - Domain Controller
```
Comment (chris-rock) [inline chef_master/source/compliance.rst]: Maybe just `CIS Microsoft Windows Server 2012 Benchmark Level 1 - Domain Controller`

**Review by chris-rock — COMMENTED** [review]
I added some minor comments, but this looks great

**Comment by kagarmoe** [comment]
gif-keyboard-922575869955760516

**Pull request merged**